### ISSUE TITLE
Develop an E4L EQ device to equalize ambisonics signals

### ISSUE THREAD

**Issue by mcslee** [EnvelopSound/EnvelopForLive#41]
Would be a chain device that takes an already-encoded surround signal, for instance output from the B-format sampler, and applies equalization to all the channels.

**Comment by compassionav** [comment]
Would be awesome to take inspiration from Fab-Filter's Pro-Q 3 in terms of design/function/capabilities!

**Comment by Landscape4** [comment]
Yes I think there is a lot of us waiting for that, specially when using a configuration with no identical loud speakers. It should be great.

**Comment by this1self** [comment]
+1 For a an E4L Spatial Equalizer! This would be awesome and will elevate the already amazing E4L Tools!
